File: C++/get.cpp
// POCO requests these headers. For HTTP requests.
#include <Poco/Net/HTTPClientSession.h>
#include <Poco/Net/HTTPRequest.h>
#include <Poco/Net/HTTPResponse.h>
#include <Poco/StreamCopier.h>
#include <Poco/Path.h>
#include <Poco/URI.h>
#include <Poco/Exception.h>

#include <iostream>
#include <string>

using namespace Poco::Net;
using namespace Poco;
using namespace std;

int main()
{
	string email, password, title, URL;

	cout<<"iSENSE GET Request Test \n";

	URL = "http://rsense-dev.cs.uml.edu/api/v1/projects/821";

	// Preparing the session.
	URI uri(URL);
	HTTPClientSession session(uri.getHost(), uri.getPort());

	// Prepare the path.
    string path(uri.getPathAndQuery());
    if (path.empty()) path = "/";

    // Send Request
    HTTPRequest req(HTTPRequest::HTTP_GET, path, HTTPMessage::HTTP_1_1);
    session.sendRequest(req);

    // Get Response
    HTTPResponse res;
    cout<<res.getStatus()<<" "<<res.getReason()<<endl;

    // Print Response
    istream &is = session.receiveResponse(res);
    StreamCopier::copyStream(is, cout);

	return 0;
}

File: HTML5/SimpleGet/Get.js
URL = 'http://rsense-dev.cs.uml.edu/api/v1/projects/683';
response = $.ajax({ type: "GET",
                url: URL,
                async: false,
                dataType: "JSON"
                }).responseText;
arg = JSON.parse(response);
=======
totalBases = [];

function get(){
    URL = 'http://rsense-dev.cs.uml.edu/api/v1/data_sets/1190?recur=true.json';
    response = $.ajax({ type: "GET",
                        url: URL,
                        async: false,
                        dataType: "JSON"
                    }).responseText;

    arg = JSON.parse(response);

    for (var i=0;i<30;i++) {
        totalBases.push(arg.data[i][647]) ;
    }

    console.log(totalBases);
}
